test/FunctionalTests/Microsoft.OData.Core.Tests/Json/JsonValueUtilsAsyncTests.cs

```diff
@@ -298,29 +328,30 @@ public async Task WriteEmptyByteShouldWork()
         {
             var byteArray = new byte[] { };
             await JsonValueUtils.WriteValueAsync(this.writer, byteArray, this.buffer);
-            Assert.Equal("\"\"", this.StreamToString());
+            Assert.Equal("\"\"", await this.StreamToString());
         }
 
         [Fact]
         public async Task WriteNullByteShouldWork()
         {
             await JsonValueUtils.WriteValueAsync(this.writer, (byte[])null, this.buffer);
-            Assert.Equal("null", this.StreamToString());
+            Assert.Equal("null", await this.StreamToString());
         }
 
         private void Reset()
         {
             this.buffer = new Ref<char[]>();
-            this.stream = new MemoryStream();
+            this.stream = new AsyncStream(new MemoryStream());
             StreamWriter innerWriter = new StreamWriter(this.stream);
-            innerWriter.AutoFlush = true;
             this.writer = new NonIndentedTextWriter(innerWriter);
         }
 
-        private string StreamToString()
+        private async Task<string> StreamToString()

```

Comment:
is it better to add a suffix "Async" for the async method name?

Reply:
Sure